config.js

```diff
@@ -0,0 +1,22 @@
+var redis = require('redis');
+
+exports.config = {
+  redis_server_url: "127.0.0.1",
+  redis_server_port: 6379,
+  redis_server_requires_auth: false,
+  redis_server_password: ""
+};
```

Comment:
These should take environment variables if they're defined so that people don't have to check in their config. For example: `redis_server_url: process.env.REDIS_SERVER_URL || "127.0.0.1",` etc..